@navikt/core/css/tooltip.css

```diff
@@ -0,0 +1,122 @@
+:root {
+  --navds-tooltip-color-text: var(--navds-semantic-color-text-inverted);
+  --navds-tooltip-color-text-inverted: var(--navds-semantic-color-text);
+  --navds-tooltip-color-background: var(
+    --navds-semantic-color-component-background-inverted
+  );
+  --navds-tooltip-color-background-inverted: var(
+    --navds-semantic-color-component-background-light
+  );
+  --navds-tooltip-color-background-key: var(--navds-global-color-gray-200);
+}
+
+@keyframes slideDown {
+  0% {
+    opacity: 0;
+    transform: translateY(-5px);
+  }
+  50% {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+@keyframes slideUp {
+  0% {
+    opacity: 0;
+    transform: translateY(5px);
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+@keyframes slideRight {
+  0% {
+    opacity: 0;
+    transform: translateX(-5px);
+  }
+  100% {
+    opacity: 1;
+    transform: translateX(0);
+  }
+}
+
+@keyframes slideLeft {
+  0% {
+    opacity: 0;
+    transform: translateX(5px);
+  }
+  100% {
+    opacity: 1;
+    transform: translateX(0);
+  }
+}
+
+.navds-tooltip {
+  background-color: var(--navds-tooltip-color-background);
+  color: var(--navds-tooltip-color-text);
+  border-radius: 4px;
+  padding: var(--navds-spacing-1) var(--navds-spacing-2);
+  white-space: nowrap;
+  animation-timing-function: cubic-bezier(0.16, 1, 0.3, 1);
+  animation-duration: 0.3s;
+  display: flex;
+  gap: 0.75rem;
+  align-items: center;
+  filter: drop-shadow(0 2px 4px var(--navds-global-color-gray-300));
+}
+
+.navds-tooltip[data-side="top"] {
+  animation-name: slideUp;
```

Comment:
Min personlige mening er at jeg ikke opplever at disse animasjonene tilfører noe annet enn at det er et forstyrrende element. Er de en del av designet? Det begynner kanskje å bli på tide å definere en animasjons-strategi for designsystemet.

Reply:
Er en del av designet. Har pratet med designer og justert de litt ned fra 300ms -> 150ms da det er litt mer snappy og mindre forstyrrende. Enig i at Animasjoner er noe vi bør få sett på etterhvert, da spesielt timing tokens og animasjonslengde.